tests/src/Flyvemdm/Tests/CommonTestCase.php

```diff
@@ -144,4 +144,17 @@ protected function agentFromInvitation(
 
       return $this->enrollFromInvitation($user, $input);
    }
+
+   /**
+    * Prepares the dependency container
+    * @param array $definition the definitions of the DI
+    */
+   public function setupDiContainer($definition) {
+      global $pluginFlyvemdmContainer;
+
+      $containerBuilder = new \DI\ContainerBuilder();
+      $containerBuilder->useAnnotations(true);
```

Comment:
I'm not in favor of use annotations and I didn't see that you aren't using it. 

We must start thinking as services for future versions and PHP-DI recommends do not use annotation for injecting dependencies on services ([ref: best practices.](http://php-di.org/doc/best-practices.html#writing-services))

Reply:
It is enabled, but not used. I followed the configuration of PHP-DI in the PR for GLPI.

Reply:
I agree about services, but this will not help us enough to write better tests. I want to use DI to access the $DB object, but we cannot solve this in the plugin only. We need DI in GLPI (because of code in CommonDBTM)